test/intl402/Segmenter/constructor/constructor/options-valid-combinations.js

```diff
@@ -7,16 +7,12 @@ description: Checks handling of valid values for the granularity option to the S
 info: |
     Intl.Segmenter ([ locales [ , options ]])
 
-    9. Let lineBreakStyle be ? GetOption(options, "lineBreakStyle", "string", « "strict", "normal", "loose" », "normal").
-    13. Let granularity be ? GetOption(options, "granularity", "string", « "grapheme", "word", "sentence", "line" », "grapheme").
+    13. Let granularity be ? GetOption(options, "granularity", "string", « "grapheme", "word", "sentence" », "grapheme").
```

Comment:
Are the line numbers still the same, or do they need to be updated?

Reply:
DONE